oci/archive/oci_src.go

```diff
@@ -28,6 +28,16 @@ func (e ImageNotFoundError) Error() string {
 	return fmt.Sprintf("no descriptor found for reference %q", e.ref.image)
 }
 
+// ImageStructureNotFoundError occurs when the archive file does not exist.
+type ImageStructureNotFoundError struct {
+	ref ociArchiveReference
+}
+
+func (e ImageStructureNotFoundError) Error() string {
+	return fmt.Sprintf("no archive file present for reference %q", e.ref.image)
```

Comment:
This should include `ref.resolvedFile` (and possibly `ref.file`), the thing that is missing.